pom.xml

```diff
@@ -271,12 +276,6 @@
 			<version>${objenesis.version}</version>
 		</dependency>
 
-		<dependency>
-			<groupId>org.reflections</groupId>
-			<artifactId>reflections</artifactId>
-			<version>0.9.12</version>
-		</dependency>
```

Comment:
The motivation behind using Reflections AND GlassGraph, was that Reflections would only see files in the local project; nothing on the extended classpath from Maven dependencies would be included.

How does one configure the current ClassGraph to only see files that are physically present on disk in the current project?

Or, to ask another way:

With this code change, if there's a dependency of the project that includes a file like the `ScriptToTest.groovy` that's added here, will these changes detect _that_ script, or not?

They should _not_.

Is that done with the `.disableJarScanning()` here: https://github.com/ExpediaGroup/jenkins-spock/pull/111/files#diff-7cf77f094673fdcead882d1c0b654a186428288727ba01a59de640f2a9f80d7fR57 ?